tests/test_x_mission_costs.py

```diff
@@ -6,30 +6,37 @@
 import pytest
 import numpy as np
 
-# from rex.utilities.loggers import LOGGERS
+from reV.handlers.exclusions import ExclusionLayers
 
-# from reVX import TESTDATADIR
-from reVX.x_mission_paths.multipliers import CostMultiplier
-from reVX.x_mission_paths.path_finder import PathFinder
+from reVX import TESTDATADIR
+from reVX.least_cost_xmission.cost_creator import XmissionCostCreator, \
+    XmissionConfig
+from reVX.least_cost_xmission.path_finder import PathFinder
+from reVX.least_cost_xmission.config import NLCD_LAND_USE_CLASSES, CELL_SIZE, \
+    TEST_DEFAULT_MULTS
+
+RI_DATA_DIR = os.path.join(TESTDATADIR, 'ri_exclusions')
+INPUT_H5F = os.path.join(RI_DATA_DIR, 'ri_exclusions.h5')
+ISO_REGIONS_F = os.path.join(RI_DATA_DIR, 'ri_iso_regions.tif')
 
 
 def test_path_cost():
     """ Test calulating path cost"""
-    costs = np.array([ [1,1,1,1,1,1], [2,2,2,2,2,2], [3,3,3,3,3,3],
+    costs = np.array([[1,1,1,1,1,1], [2,2,2,2,2,2], [3,3,3,3,3,3],
                       [2,2,2,2,2,2], [1,1,1,1,1,1], [5,5,5,5,5,5], ])
     i1 = [(0, 0), (0, 1), (0, 2), (0, 3), (0, 4), (0, 5), (1, 5), (2, 5),
           (3, 5), (4, 5), (5, 5)]
     i2 = [(0, 0), (0, 1), (1, 2), (2, 2), (2, 3), (3, 4), (4, 5), (5, 5)]
     assert round(PathFinder._calc_path_cost(costs, i1), 5) == 16.0
-    assert round(PathFinder._calc_path_cost(costs, i2), 5) ==  17.27817
+    assert round(PathFinder._calc_path_cost(costs, i2), 5) == 17.27817
 
 
 def test_land_use_multiplier():
     """ Test land use multiplier creation """
     lu_mults = {'forest': 1.63, 'wetland': 1.5}
     arr = np.array([[[0, 95, 90], [42, 41, 15]]])
-    cm = CostMultiplier()
-    out = cm._create_land_use_mult(arr, lu_mults)
+    xcc = XmissionCostCreator(INPUT_H5F, ISO_REGIONS_F, {})
```

Comment:
whats up with the empty dict here?

Reply:
iso region dict. It's not needed for the test.

Reply:
Ah! Should we make that a kwarg then?